.github/workflows/build.yml

```
@@ -174,3 +168,30 @@ jobs:
           tests/results
           tests/logs
       if: ${{ failure() }}
+
+  publish:
+    needs: test-e2e
```

I think this should be dependent on all tests passing, eg:
```suggestion
    needs: [test-e2e, standard-config-tests, default-config-tests]
```